examples/r2i/classification.cc

```diff
@@ -0,0 +1,328 @@
+/* Copyright (C) 2018-2020 RidgeRun, LLC (http://www.ridgerun.com)
+ * All Rights Reserved.
+ *
+ * The contents of this software are proprietary and confidential to RidgeRun,
+ * LLC.  No part of this program may be photocopied, reproduced or translated
+ * into another programming language without prior written consent of
+ * RidgeRun, LLC.  The user is free to modify the source code after obtaining
+ * a software license from RidgeRun.  All source code changes must be provided
+ * back to RidgeRun without any encumbrance.
+*/
+
+#include <algorithm>
+#include <getopt.h>
+#include <iostream>
+#include <memory>
+#include <string>
+#include <r2i/r2i.h>
+
+#define WIDTH_OFFSET 0
```

Comment:
Preprocessing module returns sizes tuple, width first, height second. I use this macros to avoid magic numbers. And you can see I'm using the std::get based on the offset stated by the macro.